pkg/app/appcommon/log_store.go

```diff
@@ -24,7 +29,8 @@ func NewProcLogger(conf ProcConfig) (*logging.MasterLogger, LogStore) {
 
 	log := logging.NewMasterLogger()
 	log.Logger.Formatter.(*logging.TextFormatter).TimestampFormat = time.RFC3339Nano
```

Comment:
It's not a part of your PR, but I think we could change `time.RFC3339Nano` to `timeLayout` here.